src/components/layout/Finishable/__specs__/FinishableContainer.spec.js

```diff
@@ -76,18 +103,20 @@ describe('components | FinishableContainer', () => {
         const props = mapStateToProps(state, ownProps)
 
         // then
-        expect(props).toStrictEqual({
-          offerCanBeBooked: true,
-        })
+        expect(props.offerCanBeBooked).toBe(true)
       })
     })
 
     describe('when coming from other routes', () => {
       it('should return false when offer is not bookable', () => {
         // given
+        getIsBooked.mockReturnValue(false)
+
         const state = {
           data: {
+            bookings: [{ id: 'A1', stockId: 'B1' }],
             offers: [{ id: 'A1', isBookable: false }],
```

Comment:
```suggestion
            offers: [{ id: 'C1', isBookable: false }],
```
Penser à le modifier aussi dans les `ownProps` qui suivent

Reply:
👍